private/bufpkg/bufimage/build_image.go

```diff
@@ -428,3 +425,19 @@ type buildImageOptions struct {
 func newBuildImageOptions() *buildImageOptions {
 	return &buildImageOptions{}
 }
+
+type resolverFromBuildResult struct {
```

Comment:
`resolverForBuildResult`

Needs constructor `newResolverForBuildResult`, which should be called above

Reply:
Done